xbmc/addons/AddonInstaller.h

```diff
@@ -270,15 +270,19 @@ class CAddonInstallJob : public CFileOperationJob
 class CAddonUnInstallJob : public CFileOperationJob
 {
 public:
-  CAddonUnInstallJob(const ADDON::AddonPtr &addon, bool removeData);
+  CAddonUnInstallJob(const ADDON::AddonPtr& addon,
+                     bool removeData,
+                     bool removeOrphanedDependencies);
```

Comment:
Argh. Another new bool param which leads to unreadable code at the calling side.

Reply:
thought the same exactly, will switch to an enum

Reply:
done